gatsby-node.js

```diff
@@ -139,7 +139,7 @@ exports.createPages = async ({ graphql, actions }) => {
     if (err) {
       return console.log(err);
     }
-    console.log("Directory created successfully!");
+    return console.log("Directory created successfully!");
```

Comment:
What was the error reported here? Returning `console.log` looks ... weird.

Reply:
The finding was that arrow functions need a return and since we return a console log in case there was an error, I thought we can also return one if there was no error :woman_shrugging: